tools/topology/topology2/cavs/cavs-nocodec.conf

```diff
@@ -27,6 +27,10 @@
 <dai.conf>
 <host.conf>
 
+Define {
+	MCLK 38400000
+}
+
```

Comment:
Could that define be a property of the platform and included, rather than added in CMakeLists?
We've got to reduce the number of options in CMakeLists, this should be used for board variants and not for silicon variants IMHO.

@aiChaoSONG @ranj063 @lgirdwood FYI